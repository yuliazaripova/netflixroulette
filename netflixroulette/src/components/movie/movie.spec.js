import * as React from 'react';

import Movie from './movie';

describe("Movie component", () => {
  it("should render Movie component", () => {
    const testData = {
      title: "test title",
      poster_path: "test poster",
      genres: ["test genre1", "test genre2"],
      release_date: "2017-12-13"
    }
    const component = shallow(<Movie data={testData}/>);
    expect(component).toMatchSnapshot();
  })
})