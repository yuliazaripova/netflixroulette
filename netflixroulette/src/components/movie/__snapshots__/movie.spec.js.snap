// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie component should render Movie component 1`] = `
<div
  className="movie"
  onClick={[Function]}
>
  <Poster
    classes="movie_poster"
    poster_path="test poster"
    title="test title"
  />
  <div
    className="movie_details"
  >
    <p
      className="movie_title"
    >
      test title
    </p>
    <p
      className="movie_date"
    >
      2017
    </p>
  </div>
  <p
    className="movie_genres"
  >
    test genre1 & test genre2
  </p>
</div>
`;
